draft-nygren-httpbis-httpssvc.md

```diff
@@ -830,3 +830,27 @@ as MX, SRV, and others all have their own variations here.
 Some other similar mechanisms such as SRV have a weight in-addition
 to priority.  That is excluded here for simplicity.  It could always be
 added as an optional Alt-Svc attribute.
+
+## Special cases for SvcDomainName
+
+When SvcRecordType=1, there are some special cases to handle
+for SvcDomainName:
+* How to indicate it should be equivalent to the RRNAME?
+* How to indicate it should be equivalent to the hostname?
+* What happens if it has the value "."?  SRV and MX have this
```

Comment:
I think we should resolve this before merging.  If SRV and MX special-case ".", I think that we can do the same.

I don't think we need a way to say "equivalent to the hostname".  The hostname has already been delegated to this RRNAME.